content/docs/command-reference/checkout.md

```diff
@@ -10,21 +10,26 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-`.dvc` and `dvc.lock` [files](/doc/user-guide/dvc-files-and-directories) act as
-pointers to specific version of data files or directories tracked by DVC. This
-command synchronizes the workspace data with the versions specified in the
-current `.dvc` and `dvc.lock` files.
+Synchronizes the workspace data with all the
+[tracked](/doc/command-reference/add) files and directories specified in the
+present `dvc.lock` and `.dvc` files, which act as pointers to corresponding
+<abbr>cached</abbr> data contents.
+
+The `targets` given to this command (if any) limit which files to checkout. It
```

Comment:
to checkout -> to synchronize (if we use the term in the previous paragraph)?

Reply:
No longer using "Synchronizes".